posthog/models/event/query_event_list.py

```diff
@@ -168,7 +216,10 @@ def query_events_list(
                 fragment = fragment[1:]
             order_by_list.append(translate_hogql(fragment, hogql_context) + " " + order_direction)
     else:
-        order_by_list.append(select_columns[0] + " ASC")
+        if "count(*)" in select_columns:
```

Comment:
Is `count()` a valid option too?

Reply:
Not right now, as this is the generated output SQL where `count()` hogql -> `count(*)` clickhouse sql... but since `count()` is also valid clickhouse SQL, and we might accidentally change this down the line, I added both formats here just to future proof.